core/domain/user_services.py

```diff
@@ -1565,9 +1565,9 @@ def get_last_week_dashboard_stats(user_id):
         user_id: str. The unique ID of the user.
 
     Returns:
-        list(dict): The weekly dashboard stats for the given user. Each dict
-        in the list denotes dashboard stats of the user, keyed by a datetime
-        string. If the user doesn't exist, then this function returns None.
+        dict: The dict denotes last week dashboard stats of the user, keyed by a
```

Comment:
`dict or None. The dict denotes ...`

/cc @oppia/dev-workflow-team this should be a lint check.

Reply:
done

Reply:
Sorry for the delay, We have an issue regarding this already #6095.
Thanks!